usaspending_api/download/management/commands/populate_monthly_delta_files.py

```diff
@@ -161,7 +164,17 @@ def create_local_file(self, award_type, source, agency_code, generate_since):
 
         # Create a unique temporary file with the raw query
         raw_quoted_query = generate_raw_quoted_query(source.row_emitter(None))  # None requests all headers
-        csv_query_annotated = self.apply_annotations_to_sql(raw_quoted_query, source.human_names)
+
+        # The raw query is a union of two other queries, each in parentheses. To do replacement we need to split out
+        # each query, apply annotations to each of those, then recombine in a UNION
+        csv_query_annotated = (
```

Comment:
Nothing wrong with this. Personal preference would be to define the strings and fill in strings as so:
```python
upsert_transactions = apply_annotations_to_sql(_top_level_split(raw_quoted_query, "UNION")[0].strip()[1:-1], source.human_names)
manual_deltas = apply_annotations_to_sql(_top_level_split(raw_quoted_query, "UNION")[1].strip()[1:-1], source.human_names)
csv_query_annotated = f"({upsert_transactions}) UNION ({manual_deltas})"
```